modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v8_8_2/ObjectFolderGuestPermissionUpgradeProcess.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.upgrade.v8_8_2;
+
+import com.liferay.object.constants.ObjectFolderConstants;
+import com.liferay.object.model.ObjectFolder;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.model.ResourceConstants;
+import com.liferay.portal.kernel.model.Role;
+import com.liferay.portal.kernel.model.role.RoleConstants;
+import com.liferay.portal.kernel.security.permission.ActionKeys;
+import com.liferay.portal.kernel.service.CompanyLocalService;
+import com.liferay.portal.kernel.service.ResourcePermissionLocalService;
+import com.liferay.portal.kernel.service.RoleLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Guilherme Sa
+ */
+public class ObjectFolderGuestPermissionUpgradeProcess extends UpgradeProcess {
```

Comment:
@G-Sa please simplify to `ObjectFolderUpgradeProcess`. We usually follow the `entityName`+ `UpgradeProcess` pattern.